codelabs/template.md

```diff
@@ -0,0 +1,56 @@
+author: YOUR NAME
+summary: CODELAB SUMMARY
+id: TEMPLATE-CODELAB-ID
+categories: codelab,blockly,MORE CATEGORIES
+status: Draft
+Feedback Link: https://github.com/google/blockly-samples/issues/new
+
+# CODELAB TITLE
+
+## Codelab overview
+
+### What you'll learn
+Describe the skills or tools the developers will learn by the end of this codelab.
+
+### What you'll build
+Describe what the finished product will look like.  Include screenshots or gifs.
+
+### Prerequisites
+List any terms, skills, or tools the developer should be familiar with when starting this codelab.
+
+## Setup
+
+In this section, tell the developer how to get set up to write and run their code.
+
+## Step One
```

Comment:
Is this going to be the text the user sees on the left hand side of the screen? If so, could we have some indicator that we are suppose to change this to a meaningful title?

Reply:
Changed to "Step One Title"